anaconda_project/plugins.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2016, Continuum Analytics, Inc. All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+"""Plugins assets."""
+import os
+from os.path import join
+from anaconda_project.project_commands import (_ArgsTransformer, ProjectCommand, _http_specs)
+
+from entrypoints import get_group_named
+
+
+def get_all(entry_points_group='anaconda_project.plugins'):
+    """Return all the plugins registered."""
+    return {name: plugin.load() for name, plugin in sorted(get_group_named(entry_points_group).items())}
```

Comment:
This should probably be cached since it's called on every project load; project load can happen repeatedly (like every time someone types something in an editor when editing the project file).